modules/shared/src/main/scala/org/tessellation/security/signature/Signed.scala

```diff
@@ -35,6 +35,7 @@ import org.scalacheck.Arbitrary.arbitrary
 case class Signed[+A](value: A, proofs: NonEmptySet[SignatureProof])
 
 object Signed {
+  // TODO what part of signed can we write to the log for this error?
```

Comment:
Not sure if with the current input there is something viable to be put in the error message. Please remove the TODO before the final version of this PR is ready (we tend to create tickets instead of TODOs).

Reply:
@TheMMaciek I wondered... so if there isn't anything we can print then maybe the arg has no value?

Reply:
Depends how it's used, maybe some handling logic uses that value?